cars/v1/vanilla/templates/config/jvm.options

```diff
@@ -55,6 +55,11 @@
 -XX:+UseParallelGC
 {%- endif %}
 
+{%- if use_zgc is defined and use_zgc == 'true' %}
+-XX:+UnlockExperimentalVMOptions
```

Comment:
Can you please add a comment mentioning that G1GC is no longer experimental as of JDK 15, but we need it before that? https://malloc.se/blog/zgc-jdk17